Packs/CheckPointHarmonyEndpoint/Integrations/CheckPointHarmonyEndpoint/CheckPointHarmonyEndpoint.py

```diff
@@ -0,0 +1,2703 @@
+    set_integration_context({"job_id": None, "remediation_operation_id": None})
+
+
+def convert_unix_to_date_string(unix_timestamp):
+    timestamp_in_seconds = unix_timestamp / 1000
+    date_time = datetime.fromtimestamp(timestamp_in_seconds, tz=timezone.utc)
+    return date_time.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z"
+
+def main() -> None:
+
+    params: Dict[str, Any] = demisto.params()
+    args: Dict[str, Any] = demisto.args()
+    base_url = params.get("base_url", "")
+    client_id = dict_safe_get(params, ["credentials", "identifier"])
+    secret_key = dict_safe_get(params, ["credentials", "password"])
+
+    verify_certificate: bool = not params.get("insecure", False)
+    proxy = params.get("proxy", False)
+
+    command = demisto.command()
+    demisto.debug(f"Command being called is {command}")
+
+    try:
+
+        client: Client = Client(
+            base_url=base_url,
+            client_id=client_id,
+            client_secret=secret_key,
+            verify_certificate=verify_certificate,
+            proxy=proxy,
+        )
+        client.get_token()
+        client.login()
+
+        commands = {
+            "harmony-ep-job-status-get": job_status_get_command,
+            "harmony-ep-ioc-list": ioc_list_command,
+            "harmony-ep-ioc-update": ioc_update_command,
+            "harmony-ep-ioc-create": ioc_create_command,
+            "harmony-ep-ioc-delete": ioc_delete_command,
+            "harmony-ep-policy-rule-assignments-get": rule_assignments_get_command,
+            "harmony-ep-policy-rule-assignments-add": rule_assignments_add_command,
+            "harmony-ep-policy-rule-assignments-remove": rule_assignments_remove_command,
+            "harmony-ep-policy-rule-install": rule_policy_install_command,
+            "harmony-ep-policy-rule-modifications-get": rule_modifications_get_command,
+            "harmony-ep-policy-rule-metadata-list": rule_metadata_list_command,
+            "harmony-ep-push-operation-status-list": push_operation_status_list_command,
+            "harmony-ep-push-operation-get": push_operation_get_command,
+            "harmony-ep-push-operation-abort": push_operation_abort_command,
+            "harmony-ep-anti-malware-scan": anti_malware_scan_command,
+            "harmony-ep-anti-malware-update": anti_malware_update_command,
+            "harmony-ep-anti-malware-restore": anti_malware_restore_command,
+            "harmony-ep-forensics-indicator-analyze": indicator_analyze_command,
+            "harmony-ep-forensics-file-quarantine": file_quarantine_command,
+            "harmony-ep-forensics-file-restore": file_restore_command,
+            "harmony-ep-remediation-computer-isolate": remediation_computer_isolate_command,
+            "harmony-ep-remediation-computer-deisolate": remediation_computer_deisolate_command,
+            "harmony-ep-computer-list": computer_list_command,
+            "harmony-ep-agent-computer-restart": computer_reset_command,
```

Comment:
Why do we name it `restart` and not `reset`?

Reply:
according to design, we decide to call the command 'harmony-ep-agent-computer-restart'. I will update the function name as well.